crates/ruff_linter/src/rules/flake8_executable/rules/shebang_not_executable.rs

```diff
@@ -22,12 +22,13 @@ use crate::rules::flake8_executable::helpers::is_executable;
 /// executable. If a file contains a shebang but is not executable, then the
 /// shebang is misleading, or the file is missing the executable bit.
 ///
-/// If the file is meant to be executable, add a shebang, as in:
+/// If the file is meant to be executable, add add the executable bit to the file
+/// (e.g., `chmod +x __main__.py` or `git update-index --chmod=+x __main__.py`).
 /// ```python
 /// #!/usr/bin/env python
 /// ```
 ///
-/// Otherwise, remove the executable bit from the file (e.g., `chmod -x __main__.py`).
+/// Otherwise, remove the shebang.
```

Comment:
Let's remove the code block as it's irrelevant now.

Reply:
oops, yeah, that should be removed